src/devhub/page/addon.jsx

```diff
@@ -94,6 +102,11 @@ const checkFullyRefactored = (addon_id) => {
 
 const isFullyRefactored = checkFullyRefactored(addon.addon_id);
 
+function updateWidgetEndpoint(widgetSrc) {
```

Comment:
this does not work you need to save the result from the first replace in a variable

Reply:
yeah I did that, didn't push the change yet
I will remove this after the changes are complete, will add a comment to change the contract data itself